bin/runtime-common/src/extensions/mod.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Bridge-specific transaction extensions.
+
+pub mod check_obsolete_extension;
+pub mod priority_calculator;
+pub mod refund_relayer_extension;
```

Comment:
Nit: If possible, I would move this part (related to splitting the logic to separate files) and any cosmetic changes to a separate PR. We could merge it faster, and the rest of the changes would be easier to review.

Reply:
True. Given that I'll be moving this PR and rebasing to `polkadot-sdk` anyway (where this signed extension refactoring stuff has been reverted already), I could do that first

Reply:
Thank you !